docs/devcontainers/create-dev-container.md

```diff
@@ -256,12 +256,12 @@ To get started quickly, **open the folder** you want to work with in VS Code and
 
 ![Select Docker Compose File](images/create-dev-container/select-docker-compose.png)
 
-You'll be asked to either select an existing Docker Compose file (if one exists), or pick a pre-defined container configuration from the [vscode-dev-containers repository](https://github.com/microsoft/vscode-dev-containers) in a filterable list sorted based on your folder's contents. Many of these "dev container definitions" use a Dockerfile, so select one of the following definitions as a starting point for Docker Compose:
+You'll be asked to either select an existing Docker Compose file (if one exists), or pick a pre-defined container configuration from our [first-party and community index](https://containers.dev/templates) in a filterable list sorted based on your folder's contents. You may choose to select one of the following Templates as a starting point for Docker Compose:
 
-* [Existing Docker Compose](https://aka.ms/vscode-remote/samples/existing-docker-compose)
-* [Node.js & MongoDB](https://aka.ms/vscode-remote/samples/node-mongo)
-* [Python & PostgreSQL](https://aka.ms/vscode-remote/samples/python-postgres)
-* [Docker-from-Docker Compose](https://aka.ms/vscode-remote/samples/docker-from-docker-compose).
+* [Existing Docker Compose](https://github.com/devcontainers/templates/tree/main/src/docker-existing-docker-compose)
+* [Node.js & MongoDB](https://github.com/devcontainers/templates/tree/main/src/javascript-node-mongo)
+* [Python & PostgreSQL](https://github.com/devcontainers/templates/tree/main/src/postgres)
+* [Docker-from-Docker Compose](https://github.com/devcontainers/templates/tree/main/src/docker-from-docker-compose).
```

Comment:
Ah this is an interesting one. Templates really no longer work "copy-and-paste" style. Maybe we mention the templates in the UX instead?

Reply:
I updated this section to point users to the UX, along with removing an image and description about Docker Compose that I believe to now be outdated. Let me know what you think.